character-fx.js

```diff
@@ -246,6 +247,25 @@ class CharacterFx {
       }
     };
     _updateSonicBoomMesh();
+
+    const _updateHealEffectMesh = () => {
+      if (!this.healEffect ) {
+        this.healEffect = metaversefile.createApp();
+        (async () => {
+          await metaverseModules.waitForLoad();
+          const {modules} = metaversefile.useDefaultModules();
+          const m = modules['healEffect'];
+          await this.healEffect.addModule(m);
+        })();
+        sceneLowPriority.add(this.healEffect);
+      }
+      if(this.player.hasAction('cure')){
+        this.healEffect.playEffect(this.player);

```

Comment:
I think the heal effect should be responsible for playing the animation, not this code.

To bind to the correct player, we can pass the player Id as a component so the app can look it up in the API.